executor/api/grpc/tensorflow/client.go

```diff
@@ -64,10 +64,10 @@ func (s *TensorflowGrpcClient) getConnection(host string, port int32, modelName
 func (s *TensorflowGrpcClient) Chain(ctx context.Context, modelName string, msg payload.SeldonPayload) (payload.SeldonPayload, error) {
 	switch v := msg.GetPayload().(type) {
 	case *serving.PredictRequest, *serving.ClassificationRequest, *serving.MultiInferenceRequest:
-		s.Log.Info("Identity chain")
+		s.Log.V(1).Info("Identity chain")
```

Comment:
Wouldn't it be a good idea to have integer representing log level configurable?

Reply:
Like via some environmental variable?

Reply:
Or should it be bound to the value of `debug`:
- debug = true -> V(1)
- debug = false -> V(0)
?

Reply:
I'm not sure I follow.

That line just logs that message as a debug message. It's equivalent to `logger.Debug("IdentityChain")`.

Reply:
So basically `"Identity chain` is always `Debug` level. 
Yeah, ok, makes sense. Somehow I thought it may work other way around that here we could be setting level at which it is reported but it makes sense now. nvm.